### ISSUE TITLE
problem with train without lstmf

### ISSUE THREAD

**Issue by josef821** [tesseract-ocr/tesseract#4282]
### Current Behavior

i try to train without lstmf in tesseract 5.4.1
but i got this error:

Num outputs,weights in Series:
  1,36,0,1:1, 0
Num outputs,weights in Series:
  C3,3:9, 0
  Ft16:16, 160
Total weights = 160
  [C3,3Ft16]:16, 160
  Mp3,3:16, 0
  TxyLfys48:48, 12480
  Lfx96:96, 55680
  RxLrx96:96, 74112
  Lfx192:192, 221952
  Fc179:179, 34547
Total weights = 398931
Built network:[1,36,0,1[C3,3Ft16]Mp3,3TxyLfys48Lfx96RxLrx96Lfx192Fc179] from request [1,36,0,1 Ct3,3,16 Mp3,3 Lfys48 Lfx96 Lrx96 Lfx192 O1c179]
Training parameters:
  Debug interval = 0, weights = 0.1, learning rate = 0.002, momentum=0.5
null char=2
Deserialize header failed: data/ara-ground-truth/1_530arial_fa_S16_R0.png
Deserialize header failed: data/ara-ground-truth/2_1914KTabassomKTabassom_fa_S22_R_1_4.png
Deserialize header failed: data/ara-ground-truth/3_2492MjRabie_fa_S22_R0.png
Deserialize header failed: data/ara-ground-truth/4_2910Roze_fa_S14_R0.png
Deserialize header failed: data/ara-ground-truth/5_268Aban_fa_S14_R0.png
Deserialize header failed: data/ara-ground-truth/6_2091LateefRegOT_fa_S21_R0.png
Deserialize header failed: data/ara-ground-truth/7_3272SultanAdanBold_fa_S13_R1_6.png
Deserialize header failed: data/ara-ground-truth/8_427AlRaiMedia-BoldBold_fa_S16_R1_8.png
Deserialize header failed: data/ara-ground-truth/9_1052Setareh_fa_S17_R0.png
Load of page 0 failed!
Load of images failed!!

### Expected Behavior

as [pull request 4215](https://github.com/tesseract-ocr/tesseract/pull/4215) training without lstmf and just with *.png file in train list must working

### tesseract -v

tesseract 5.4.1
 leptonica-1.82.0
  libgif 5.1.9 : libjpeg 8d (libjpeg-turbo 2.1.1) : libpng 1.6.37 : libtiff 4.3.0 : zlib 1.2.11 : libwebp 1.2.2 : libopenjp2 2.4.0
 Found AVX2
 Found AVX
 Found FMA
 Found SSE4.1
 Found libarchive 3.6.0 zlib/1.2.11 liblzma/5.2.5 bz2lib/1.0.8 liblz4/1.9.3 libzstd/1.4.8
 Found libcurl/7.81.0 OpenSSL/3.0.2 zlib/1.2.11 brotli/1.0.9 zstd/1.4.8 libidn2/2.3.2 libpsl/0.21.0 (+libidn2/2.3.2) libssh/0.9.6/openssl/zlib nghttp2/1.43.0 librtmp/2.3 OpenLDAP/2.5.16

### Other Operating System

ubuntu 20.04

### Compiler

gcc 11.4

### CPU

12700h

**Comment by josef821** [comment]
problem was solved [link](https://github.com/tesseract-ocr/tesseract/issues/2456#issuecomment-513184281)

**Comment by dejvizelo** [comment]
My list.train and list.eval consist of paths to images and I am getting a similar error after running `lstmtraining`:
```
Deserialize failed: /path/ipa_100977_font3.tif read 0/1229531648 lines
```

@josef821 Do you have an example makefile for training without lstmf files? I checked of the EOL of list.train and list.eval and they seem to be UNIX EOL.
